### DISCUSSION TITLE
How to Learn Elixir and Phoenix Guide

### DISCUSSION THREAD

**Discussion by wintermeyer** [3Stones-io/omedis#363] [General]
In yesterday's call @fradarr mentioned a potential new programmer who is in the process of learning Elixir and Phoenix. I just created a Wikipage for potential team member candidates who want to learn.

https://github.com/wintermeyer/omedis/wiki/How-to-Learn-Elixir-and-Phoenix-Framework

Please add information which is helpful for newbies. And don't be afraid to improve my text.

**Comment by almirsarajcic** [comment]
I haven't used it, but I've heard about this website too
https://www.educative.io
It has AI-powered (buzzword :D) courses, e.g. https://www.devpath.com/courses/functional-programming-elixir
